validation/snapshot/snapshot_restore_k8s_upgrade_test.go

```diff
@@ -52,35 +49,25 @@ func (s *SnapshotRestoreK8sUpgradeTestSuite) TestSnapshotRestoreK8sUpgrade() {
 
 	tests := []struct {
 		name         string
+		clusterType  string
 		etcdSnapshot *etcdsnapshot.Config
 		client       *rancher.Client
 	}{
-		{"Restore Kubernetes version and etcd", snapshotRestoreK8sVersion, s.client},
+		{"RKE1_Restore_ETCD_K8sVersion", "-rancher", snapshotRestoreK8sVersion, s.client},
```

Comment:
Same comment/question earlier of needing to provide each cluster type. Same for the rest of the snapshot tests.